src/styles/index.scss

```diff
@@ -36,9 +24,25 @@ html {
 
 		color: $text-color;
 		background-color: $body-bg;
+
+		p, span, h1, h2, h3, h4, h5, h6, div {
+			color: $text-color;
+		}
```

Comment:
@ateska
Hi, rewriting those elements here is needed b/c otherwise they may inherit text color not from body, but from some another parent container which is closer to element. E.g. `Table` from reactstrap adds to `tr` element some black font-color on hover which overrides our color from body. In order not to trace all occasions where we can find such behaviour and fix them explicitly, we decided to assign our text color to all of the elements. Such approach ofc may lead to problem like we have with `react-json-view` when our styles rewrites styles of some component which also rely on inheritance and when we don't want to rewrite those. But in case of `react-json-view` that's actually a problem of `react-json-view` itself. And in our applications we've found only one library which has such problem. BTW solution for that one we have below (lines 32-37)

Reply:
Ok. Understand.